finnhub/python/publisher.py

```diff
@@ -0,0 +1,77 @@
+import asyncio
+import json
+import os
+
+import websockets
+
+from pyensign.events import Event
+from pyensign.ensign import Ensign
+
+from utils import handle_ack, handle_nack
+
+
+class TradesPublisher:
+    """
+    TradesPublisher queries an API for trading updates and publishes events to Ensign.
+    """
+
+    def __init__(self, symbols=["AAPL", "MSFT", "AMZN"], topic="trades"):
+        self.symbols = symbols
+        self.topic = topic
+        self.ensign = Ensign()
```

Comment:
Would the `Ensign credentials` go inside `Ensign` if they're specified in the environment?

Reply:
if the client id and secret in set in the environment variables, then they don't have to be specified in the call to `Ensign()`.